components/atom/upload/src/index.scss

```diff
@@ -0,0 +1,61 @@
+@import '~@schibstedspain/sui-theme/lib/index';
+
+$bg-atom-upload: #f5f5f5 !default;
+$c-atom-upload: #979797 !default;
+
+.sui-AtomUpload {
+  align-items: center;
+  background-color: $bg-atom-upload;
+  border: 2px dashed $c-atom-upload;
+  color: $c-atom-upload;
+  display: flex;
+  flex-direction: column;
+  font-family: $ff-sans-serif;
+  justify-content: center;
+  padding: $p-l * 3 $p-l * 2 $p-l;
+  text-align: center;
+
+  &-blockText {
+    align-items: center;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+
+    h4,
```

Comment:
is this part of the wrapped component? Is there no class you could use as selector?